lib/Connection.js

```diff
@@ -201,6 +201,8 @@ class Session extends EventEmitter {
   }
 
   async dispose() {
+    if (!this._connection)
```

Comment:
Let's `console.assert` instead.

Reply:
instead of the error? that sounds reasonable